src/main/java/com/google/codeu/servlets/SettingServlet.java

```diff
@@ -0,0 +1,82 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+import java.util.Set;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Datastore;
+import com.google.codeu.data.User;
+import com.google.gson.JsonObject;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+/**
+ * Handles fetching and saving user data.
+ */
+@WebServlet("/setting")
+public class SettingServlet extends HttpServlet {
+
+    private Datastore datastore;
+
+    @Override
+    public void init() {
+        datastore = new Datastore();
+    }
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws IOException {
+
+        JsonObject jsonObject = new JsonObject();
+
+        UserService userService = UserServiceFactory.getUserService();
+
+        if (userService.isUserLoggedIn()) {
+            String userEmail = userService.getCurrentUser().getEmail();
+
+            User user = datastore.getUser("test@example.com");
+            System.out.println(userEmail);
```

Comment:
Let's remove all debugging code (also remove other System.out.println below).

Generally, you can setup logging libraries to print out stuff, so that it prints timestamp + code location + some meaningful messages + data. But you should not use System.out.println, as it usually isn't helpful to other team members.

Reply:
Removed the debugging code.